docs/proposals/proposal-workflow-authentication.md

```diff
@@ -0,0 +1,69 @@
+# Proposal for Workflow Authentication
+
+## Proposal summary
+
+**For workflow-to-workflow-service authentication, use:**
+
+* **for jobs: one-time per-job tokens**, i.e. a single one-time token for
+  every job;
```

Comment:
for the life time of the the task-job?

Reply:
Yes, good point, that's an important aspect to clarify. Thanks.